docs/resource-policy/installation.md

```diff
@@ -1 +1,264 @@
-# Installation
+# Installation methods
+
+This repository hosts a collection of plugins of various types, one of which is the resource
+policy plugins. In this example, we will demonstrate the installation process for the topology-aware
+plugin, which falls under the resource policy type. The installation methods outlined
+here can be applied to any other plugin hosted in this repository, regardless of its type.
+
+You have the option to choose either of these installation methods to install the NRI plugin into
+your cluster. 
+
+1. [Helm](#Helm)
+2. [Kustomize](#Kustomize)
+3. [Manual](#Manual)
+
+Regardless of the chosen installation method, the NRI plugin installation includes the
+following components: DaemonSet, ConfigMap, CustomResourceDefinition, and RBAC-related objects.
+
+### Prerequisites
+
+Container runtime:
+- containerD:
+    - At least [containerd 1.7.0](https://github.com/containerd/containerd/releases/tag/v1.7.0) release version to use the NRI feature
+    - Enable NRI feature by following [these](TODO link) detailed instructions.
+- CRI-O
+    - At least [v1.26.0](https://github.com/cri-o/cri-o/releases/tag/v1.26.0) release version to use the NRI feature
+    - Enable NRI feature by following [these](TODO link) detailed instructions.
+
+## Helm
+
+1. Clone the project to your local machine
+    ```sh
+    git clone https://github.com/containers/nri-plugins.git
```

Comment:
Can the helm chart installation done without this step i.e., can we point directly to a file in github?

Reply:
I'm afraid it cannot be done without a packed helm chart. I don't think we want to include packed chart (.tgz) in the main branch so this will become available with the first release when the helm repo within gh-pages gets populated.

What we could do is to extend the update-gh-pages script to always pack the chart from the tip of the main branch addi it to the gh-pages branch (and update the helm repo index). But I think that's subject of a separate PR...

Reply:
Yes, I think we need at least one release and then update the doc again.